lib/Container.js

```diff
@@ -246,6 +282,73 @@ Container.prototype.getExposedPortFromContainerInfo = function(data, port) {
 };
 
 /**
+ * Gets the disk usage of the container.
+ * Current implementation requires the AUFS Storage driver and to be run as root.
+ * Errors if there's no root access or not using the AUFS driver
+ *
+ * @param {Function} done - callback that takes err and an object as arguments. The object has {imageSize, containerSize}, for the size of the inderlying image and container layer respectively
+ */
+Container.prototype.getDiskUsage = Promise.promisify(function(done){
+  var result = {containerSize: undefined, imageSize: undefined}
+
+  if(process.getuid() !== 0){
+    return done(new Error("getDiskUsage must be run as root"), result)
+  }
+
+  var self = this
+  // stat the container to ensure that it's using the AUFS driver
+  this.container.inspect(function(err, info){
+    if(err) {
+      return done(null, result)
+    }
+
+    if(info.Driver != "aufs"){
+      return done(new Error(`Container Driver must be 'aufs', found '${info.Driver}'`), result)
+    }
+
+    containerSize(info.Id, function(err, containerSize){
+      result.containerSize = err ? err : containerSize
+      imageSize(info.Image, function(err, imageSize){
+        result.imageSize = err ? err : imageSize
+        done(null, result)
+      })
+    })
+
+    function containerSize(containerId, done){
```

Comment:
This seems like it should be a top level method on the container call by `getDiskUsage()`.